main.go

```diff
@@ -34,7 +34,7 @@ import (
 	// +kubebuilder:scaffold:imports
 )
 
-const useragent = "google-pso-tool/gke-autoneg-controller/0.9.2-dev"
+const useragent = "google-pso-tool/gke-autoneg-controller/0.9.4-dev"
```

Comment:
Let's not bump this here - we can do it as part of a release.

Reply:
Took it out of commit.